packages/contracts/scripts/generate-deployed-artifacts.js

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env node
+const path = require('path')
+const glob = require('glob')
+const fs = require('fs')
+
+;(async () => {
+  console.log(`writing deployed contract artifacts typescript file`)
+
+  let content = `
+  /* eslint-disable @typescript-eslint/no-var-requires, no-empty */
+  /*
+  THIS FILE IS AUTOMATICALLY GENERATED.
+  DO NOT EDIT.
+  */
+  `
+
+  const deploymentNames = fs
+    .readdirSync(path.resolve(__dirname, '../deployments'), {
+      withFileTypes: true,
+    })
+    .filter((entry) => {
+      return entry.isDirectory()
+    })
+    .map((entry) => {
+      return entry.name
+    })
+
+  const artifactNames = []
+  for (const deploymentName of deploymentNames) {
+    const deploymentArtifacts = glob.sync(
+      path.resolve(__dirname, '../deployments') +
+        '/' +
```

Comment:
Perhaps `path.join` as this won't work on windows

Reply:
Good point